src/MahApps.Metro/Controls/ButtonsAlignment.cs

```diff
@@ -3,6 +3,7 @@
    public enum ButtonsAlignment
    {
       Left,
-      Right
+      Right,
+      OpposingSides
```

Comment:
@MarcoMartins86 I think `Opposite` fits better with the other two values.

Reply:
Yes, you're right. Done.